packages/mobile/src/paymentRequest/IncomingPaymentRequestListItem.tsx

```diff
@@ -30,6 +29,8 @@ interface OwnProps {
 
 type Props = OwnProps & WithNamespaces
 
+const avatarSize = 40
```

Comment:
nit: We might be inconsistent on this throughout the codebase but I think it should be `AVATAR_SIZE`.